src/components/Home.js

```diff
@@ -12,15 +13,16 @@ import FriendsList from './FriendsList';
 // import { PhotosContext } from '../contexts/photos-context'
 import ContextTest from './ContextTest';
 
-const Home = () => {
+const Home = ({ logOut }) => {
+  // const value = useContext(UserInfoContext);
   // Use context in this instance is for the friends list
   return (
     <div>
       <Typography variant="h2">
         HELLO WORLD!
         <br />
-        <ContextTest />
-        <FriendsList />
+        {/* <div>{`Hello ${value}`}</div> */}
+        {/* <FriendsList friends={friends} pending={pending} requested={requested}/> */}
```

Comment:
Did you intend to remove the friend list?

Reply:
No I don't believe I did... I wonder if that was the result of fixing a merge conflict I had